src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
Lets leave the Wiggle data in add a FIXME to the methods that
they need to be resolved.

CH Albach <notifications@github.com> writes:
> On SearchAnnotations, there is one primary unresolved issue which is around
> searching 'type'. The OntologyTerm as a search parameter doesn't really make
> sense; you'd need to define some very specific rules about how matching works
> and how subfields should be populated.

This is a GA4GH-wide problem. it impacts metadata as well.
Leave it undefined for now and a FIXME.

> I guess I would prefer that there were
> just and ontology ID (as there is a reference ID) or else an enum of types
> which would yield a more natural query.

enums are great for programmers, but they just don't work for
the kind of data we have.  We have to keep this about data
exchange

I have some thoughts on address this I want to bring up in the
metadata group, which is essentially an ontology query that
gives you back a record with fields for the common terms and
values of the ontology links.  These could be ontology ids too,

> This might be a rathole though, so the
> simplest solution is to remove the search by type functionality from the
> initial PR.

Absolutely FIXME RATHOLE would be great ;-)

> Beyond that, my only other major concern is around naming, which uses an
> abbreviated style which is inconsistent with the rest of the GA4GH APIs.

Already changed, please pull.  This does bring up the need to clean up
the whole GA4GH name space, but that is beyond this scope.

I am think that with graph references in motion, we should make
this topic branch on the main and let the wider debate happen
there..  Benedict is pushing me to do this.

Mark